src/context/StarredContext.tsx

```
@@ -0,0 +1,44 @@
+  // 즐겨찾기 여부 확인 함수
+  const isStarred = (id: string) => starredItems.includes(id);
+
+  return (
+    <StarredContext.Provider value={{ starredItems, toggleStar, isStarred }}>
+      {children}
+    </StarredContext.Provider>
+  );
+};
+
+// 커스텀 훅 (App.tsx에서 useStarred를 쓰고 있으므로 필수)
+export const useStarred = () => {
```

ESLint

[error] Fast refresh only works when a file only exports components. Use a new file to share constants or functions between components.

(react-refresh/only-export-components)